web/src/main/java/se/inera/intyg/intygstjanst/web/service/dto/GetSickLeaveServiceRequest.java

```diff
@@ -41,4 +41,5 @@ public class GetSickLeaveServiceRequest {
     String protectedPersonFilterId;
     LocalDate fromSickLeaveEndDate;
     LocalDate toSickLeaveEndDate;
+    List<String> rekoStatuses;
```

Comment:
Suggest change of name to rekoStatusTypeIds. This change should be made in necessary places.